theme.php

```diff
@@ -23,10 +23,458 @@
  * Theme Information
  */
 $aTheme = array(
-    'id'           => 'flow',
-    'title'        => 'Flow',
-    'description'  => 'Flow is OXID`s official responsive theme based on the CSS framework Bootstrap 3.',
-    'thumbnail'    => 'theme.jpg',
-    'version'      => '1.0.0-beta',
-    'author'       => '<a href="http://www.oxid-esales.com" title="OXID eSales AG">OXID eSales AG</a>',
+    'id'          => 'flow',
+    'title'       => 'Flow',
+    'description' => 'Flow is OXID`s official responsive theme based on the CSS framework Bootstrap 3.',
+    'thumbnail'   => 'theme.jpg',
+    'version'     => '1.0.0-beta',
+    'author'      => '<a href="http://www.oxid-esales.com" title="OXID eSales AG">OXID eSales AG</a>',
+    'settings'    => array(
```

Comment:
Maybe this should be something like

'settings' => include 'theme-settings.php'

and all settings goes into theme-settings.php